Frontend/src/Components/Faults/FaultsView.js

```diff
@@ -285,14 +285,217 @@ export default function Faults(props) {
+  };
+
+  const getMotorControllerString = () => {
+    let motorControllerStrings = [];
+
+    // Add any motor controller status issues to the tooltip
+    // The first element pushed should be capitalized, and any additional
+    // parts of the string should not be
+    if (props.data?.["mc_status"][0] === 1) {
+      motorControllerStrings.push(motorControllerStrings.length === 0 ? "Over current (motor or battery current too high)" : "over current (motor or battery current too high)");
+    }
+    if (props.data?.["mc_status"][0] === 2) {
+      motorControllerStrings.push(motorControllerStrings.length === 0 ? "Unused error code 2, possibly 'Over current (motor or battery current too high)' or 'Hole sensor fault'" : "unused error code 2, possibly 'Over current (motor or battery current too high)' or 'Hole sensor fault'");
+    }
+    if (props.data?.["mc_status"][0] === 3) {
+      motorControllerStrings.push(motorControllerStrings.length === 0 ? "Hole sensor fault" : "hole sensor fault");
+    }
+    if (props.data?.["mc_status"][0] === 4) {
+      motorControllerStrings.push(motorControllerStrings.length === 0 ? "Motor locked" : "motor locked");
+    }
+    if (props.data?.["mc_status"][0] === 5) {
+      motorControllerStrings.push(motorControllerStrings.length === 0 ? "Sensor fault 1 (current sensor, thermistor, voltage sensor)" : "sensor fault 1 (current sensor, thermistor, voltage sensor)");
+    }
+    if (props.data?.["mc_status"][0] === 6) {
+      motorControllerStrings.push(motorControllerStrings.length === 0 ? "Sensor fault 2 (accelerator voltage out of range)" : "sensor fault 2 (accelerator voltage)");
+    }
+    if (props.data?.["mc_status"][0] === 7) {
+      motorControllerStrings.push(motorControllerStrings.length === 0 ? "Unused error code 7, possibly 'Sensor fault 2 (accelerator voltage out of range)' or 'High battery voltage'" : "unused error code 7, possibly 'Sensor fault 2 (accelerator voltage out of range)' or 'High battery voltage'");
+    }
+    if (props.data?.["mc_status"][0] === 8) {
+      motorControllerStrings.push(motorControllerStrings.length === 0 ? "High battery voltage" : "high battery voltage");
+    }
+
+    /**
+     * Generates a string correlating an overheating status to the actual temperature value and
+     * displays the action that the motor controller is taking in response
+     */
+    const getTempActionString = () => {
+      let motorControllerTemperature = props.data?.["motor_controller_temp"][0];
+      let overheatAction = "";
+      if (motorControllerTemperature > 85 && motorControllerTemperature < 95) {
+        overheatAction = "controller will reduce power to 1/2"
+      } else if (motorControllerTemperature > 95 && motorControllerTemperature < 105) {
+        overheatAction = "controller will reduce power to 1/4";
+      } else if (motorControllerTemperature > 105) {
+        overheatAction = "drive will stop";
+      } else {
+        overheatAction = "no action taken";
+      }
```

Comment:
```suggestion
      if (motorControllerTemperature > 105) {
        overheatAction = "drive will stop";
      } else if (motorControllerTemperature > 95) {
        overheatAction = "controller will reduce power to 1/4";
      } else if (motorControllerTemperature > 85) {
        overheatAction = "controller will reduce power to 1/2"
      } else {
        overheatAction = "no action taken";
      }
```
Also, double check if it's > or >=

Reply:
Done, and it's >=